usage.log

```diff
@@ -0,0 +1,5 @@
+To build project: 
+$> mvn clean install
+
+To run test:
+$> mvn -Dtest=EspnSeleniumTest2 test
```

Comment:
What does specifying the test name here do? Would `mvn test` provide the same result?